internal/support/scheduler/router.go

```diff
@@ -0,0 +1,231 @@
+//
+// Copyright (c) 2018 Tencent
+//
+// Copyright (c) 2017 Dell Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package scheduler
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/edgexfoundry/edgex-go/pkg/models"
+	mux "github.com/gorilla/mux"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+func LoadRestRoutes() http.Handler {
+	r := mux.NewRouter()
+	b := r.PathPrefix("/api/v1").Subrouter()
+
+	//ping
+	b.HandleFunc("/ping", ping).Methods(http.MethodGet)
```

Comment:
Move this above line 22 and use the internal.ApiPingRoute constant. Also, please provide the config endpoint which the SMA will need. Example:
https://github.com/edgexfoundry/edgex-go/blob/master/internal/support/logging/server.go#L211

Reply:
Updated router to use the mux style incorporated in your logging/server.go example